packages/components/src/context-actions/ContextActions.test.jsx

```diff
@@ -129,12 +129,7 @@ it('renders a promise returning menu items properly', async () => {
 
 it('renders an empty menu for a rejected promise', () => {
   const mock = DEFAULT_MOCK();
-  const promise = {
-    then: () => {},
-    catch: callback => {
-      callback();
-    },
-  };
+  const promise = new Promise((resolve, reject) => reject());
```

Comment:
You can just write this as `Promise.reject()`
```suggestion
  const promise = Promise.reject();
```